### DISCUSSION TITLE
GSOC  2024

### DISCUSSION THREAD

**Discussion by alyaa999** [polypheny/Polypheny-DB#469] [Q&A] [answered]
Hello, I'm Alyaa Mamoon, a computer science student. I'm keen on joining your organization and contributing to it, particularly participating in GSOC 2024. I'm new to contributing, but I'm eager to learn and would appreciate your guidance. I'm particularly interested in the Cypher Quality Assurance idea. Could you help me get started and provide some direction?

**Comment by hennlo** [comment, marked as answer]
@alyaa999 Hi @alyaa999 Welcome in our community: Feel free to join our [Discord Server](https://discord.gg/TC67WN5V)
to get all information around GSoC and get in touch with the rest of the community.
